Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/TotalExecutionTimeEvent.java

```diff
@@ -0,0 +1,29 @@
+package net.simon987.cubotplugin.event;
+
+import net.simon987.server.event.GameEvent;
+import net.simon987.server.game.objects.GameObject;
+
+public class TotalExecutionTimeEvent extends GameEvent {
+
+    public TotalExecutionTimeEvent(){
+
+    }
+
+    public TotalExecutionTimeEvent(GameObject object, double c) {

```

Comment:
You can confirm in CPU.java but I think that the execution time is an integer. (You can keep the double utility method in UserStats())

Reply:
I removed that part completely as I don't see a use for it